### DISCUSSION TITLE
Roborio 2 network settings - request for verification

### DISCUSSION THREAD

**Discussion by sciencewhiz** [wpilibsuite/2024Beta#63] [General Technical]
Can someone with a roboRIO 2 with the 2024 beta image verify the network settings and see if it is still dhcp only or dhcp and link local? https://docs.wpilib.org/en/latest/docs/yearly-overview/known-issues.html#roborio-2-0-ethernet-settings

**Comment by ThadHouse** [comment]
This is fixed in 2024 V2.0
